### PR TITLE
[IDWA-OPS-58] Fix OCR unit test to trigger

### PR DESCRIPTION
**Pull request by derekadombek**
# Pull Request

## Description
Before I had it setup so it was triggering on every upstream commit. To refine this more, this will trigger only after a PR is `opened`, `synchronize`, or `reopened`. `synchronize` is the type that will trigger for every commit after initial PR creation.

**NOTE:** These types are default to `on: pull_request` so theres no need to add them in the code.

## Related Issues
#58 

## Checklist
Please review and complete the following checklist before submitting your pull request:

- [ ] I have ensured that the pull request is of a manageable size, allowing it to be reviewed within a single session.
- [ ] I have reviewed my changes to ensure they are clear, concise, and well-documented.
- [ ] I have updated the documentation, if applicable.
- [ ] I have added or updated test cases to cover my changes, if applicable.
- [ ] I have minimized the number of reviewers to include only those essential for the review.
- [ ] I have notified teammates in the review thread to build awareness.

## Checklist for Reviewers
Please review and complete the following checklist during the review process:

- [ ] The code follows best practices and conventions.
- [ ] The changes implement the desired functionality or fix the reported issue.
- [ ] The tests cover the new changes and pass successfully.
- [ ] Any potential edge cases or error scenarios have been considered.

### CODE REVIEW

File: .github/workflows/ocr-tests.yml [hunk 1]
```diff
@@ -1,11 +1,6 @@
...
 on:
   pull_request:
-    types: [opened, reopened]
-    paths:
-      - .github/workflows/ocr-tests.yml
-      - OCR/**
-  push:
     paths:
       - .github/workflows/ocr-tests.yml
       - OCR/**
```
Comment (zdeveloper) [inline .github/workflows/ocr-tests.yml:6]: do we need the quotes around the paths?

**Review by zdeveloper — APPROVED** [review]
